meetings/submissions/serializers.py

```diff
@@ -41,3 +45,12 @@ def get_can_edit(self, obj):
         request = self.context.get('request')
         user = request.user
         return (user == obj.contributor or user == obj.conference.admin)
+
+    def get_download_count(self, obj):
+        conn = httplib.HTTPConnection("api.osf.io")
```

Comment:
Don't hardcode this value. Look at other places that osf is referenced and how it is handled.